vendor/licenses/git_submodule/slint-tmLanguage.dep.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: slint-tmLanguage
+version: 1ad143138d60211dde6af6d9c2ad2a2d0d626c92
+type: git_submodule
+homepage: https://github.com/slint-ui/slint-tmLanguage.git
+license: none
+licenses: []
+notices: []
```

Comment:
This should be populated with the deets of the license from this repo.

Try manually running this command https://github.com/github-linguist/linguist/blob/ca188d5b069684d5160170e2ab86fa995ca7fe0f/script/add-grammar#L134